Package.swift

```diff
@@ -52,6 +53,22 @@ let package = Package(
             ]
         ),
 
+        .target(
+            name: "TracingInstrumentation",
+            dependencies: [
+                .product(name: "Baggage", package: "swift-baggage-context"),
+                "Instrumentation"
```

Comment:
ok; do we need to spell out Baggage dependenccy explicitly tho?

Reply:
Nope, will remove 👍

Reply:
thanks